src/test/java/uk/gov/companieshouse/company/profile/api/InsolvencyApiClientServiceTest.java

```diff
@@ -45,7 +45,7 @@ void should_invoke_chs_kafka_endpoint_successfully() throws ApiErrorResponseExce
         when(privateChangedResourceHandler.postChangedResource(Mockito.any(), Mockito.any())).thenReturn(changedResourcePost);
         when(changedResourcePost.execute()).thenReturn(response);
 
-        ApiResponse<?> apiResponse = insolvencyApiService.invokeChsKafkaApi("CH4000056");
+        ApiResponse<?> apiResponse = insolvencyApiService.invokeChsKafkaApi("123456", "CH4000056");
```

Comment:
cosmetic: Maybe better change to ApiResponse<Void>

Reply:
Done